protos/action/action.proto

```diff
@@ -84,11 +160,11 @@ message SetTakeoffAltitudeResponse {
 message GetMaximumSpeedRequest {}
 message GetMaximumSpeedResponse {
     ActionResult action_result = 1;
-    float speed = 2;
+    float speed = 2; // Maximum speed (in metres/second)
```

Comment:
Not a change for this PR but we should decide on what we do about units!

Reply:
What do you mean? In terms of how we document them?

Reply:
I think they should be part of the variable name (unless we can have units that reflect it).

Reply:
Right. I don't think I have a strong opinion on that matter, but if we want to change, a good moment would be after we have the C++ auto-generation, and before we move to version 1.x :+1:.